language_model/tensorflow/bert/README.md

```diff
@@ -85,20 +90,21 @@ python run_pretraining.py \
   --iterations_per_loop=1000 \
   --max_predictions_per_seq=76 \
   --max_seq_length=512 \
-  --num_train_steps=150000 \
-  --num_warmup_steps=0 \
+  --num_train_steps=682666666 \
+  --num_warmup_steps=1562 \
   --optimizer=lamb \
-  --save_checkpoints_steps=1000 \
+  --save_checkpoints_steps=20833 \
   --start_warmup_step=0 \
-   --num_gpus=16 \
-  --train_batch_size=4/
+  --num_gpus=8 \
+  --train_batch_size=24
```

Comment:
Is the global batch size 24 or 24*8? 
And what is special about 1562 and 20833?

Reply:
The global batch size is 24.
Nothing special about 1562; it's just what's used for the reference runs.

20833 * 24 is roughly 500k, which is the interval needed to do evaluation.
I will have another PR to add the 3M start evaluation input.

Reply:
The num_gpus flags is from PR 384, with the multi-gpu parts and step counter correction.
Yes, each GPU only gets to work with batch size 3 with a 16GB-V100. Per-GPU batch size 4 will OOM. 

Thank you for reviewing the PR.

Reply:
Got it, thank you!